client/my-sites/importer/site-importer/site-importer-input-pane.jsx

```diff
@@ -374,6 +385,50 @@ class SiteImporterInputPane extends React.Component {
 		} );
 	};
 
+	renderUrlHint = () => {
+		switch ( this.props.importerData.engine ) {
+			case 'wix':
+				return (
+					<div>
+						<p>
+							{ this.props.translate( 'Please use one of following formats for the site URL:' ) }
+						</p>
+						<ul>
+							<li>
+								<span className="site-importer__site-importer-example-domain">example.com</span>
+								{ ' - ' }
```

Comment:
I haven't tested it here but from the previous work on this component, I know I had to be explicit about the white space, otherwise it would be eaten out by the autoformatted + compiler or minifier and it would be just a dash with no spaces around it.

This was the previous version: https://github.com/Automattic/wp-calypso/blob/master/client/my-sites/importer/site-importer/site-importer-input-pane.jsx#L449

I just put the whole space-dash-space into one string so it's not all over the place. Let me know if it makes sense.

Reply:
Ah right, I see what you mean. The newer version is clearer!